app/scripts/app.js

```diff
@@ -1,6 +1,10 @@
 angular.module('recsApp', ['elasticsearch']);
 
-var esBaseURL = 'http://ec2-54-234-184-186.compute-1.amazonaws.com:9616';
+//var esBaseURL = 'http://ec2-54-234-184-186.compute-1.amazonaws.com:9616';
+// We expect a entry called elastic-graph-recommender that points to the
+// Elasticsearch server, typically in your /etc/hosts file.  This is to
+// be Docker friendly.
+var esBaseURL = 'http://localhost:9200';
```

Comment:
Shiite.    There was a variable in the past there.   Thank you!   I'll fix that.